scopes/angular/angular/templates/angular-component/src/lib/component-spec.ts

```diff
@@ -1,10 +1,10 @@
 import { ComponentContext } from '@teambit/generator';
 
-export const testFile = (context: ComponentContext) => {
+export const componentSpecFile = (context: ComponentContext) => {
   const { name, namePascalCase: Name } = context;
 
   return {
-    relativePath: `${name}.spec.ts`,
+    relativePath: `src/${name}.spec.ts`,
```

Comment:
is it necessary to have each file under the `src` folder within the component?

Reply:
I guess it's related to the public_api file is outside the src?

Reply:
yes, I'm just trying to follow the style that Angular promotes (based on their style guide and what angular CLI generates), but if you have a good reason to avoid that I don't really mind